### ISSUE TITLE
An error was encountered playing AVI's

### ISSUE THREAD

**Issue by p37307** [jellyfin-archive/jellyfin-roku-legacy#650]
**Describe the bug**

Upgraded Windows 10 Jellyfin server via installer to Jellyfin 10.8.0 from 10.8Beta3 on my test server.

Played the AVI video, Pilot.avi, which is a sole video in a folder 

A popup on Jellyfin Roku says, "An error was encountered while playing this item." (This only occurs playing AVI's)

Issue occurs with "Jellyfin Roku" "1.4.12". There's no update on Roku from 1.4.12 to 1.4.13 for whatever reason.

Jellyfin server doesn't create an ffmpeg log.

Jellyfin server does send smtp email notification saying the video has ended as I have that setup in my plugin settings.

It keeps attempting to open the AVI, repeatedly and sends the emails, in a loop, unless I exit Jellyfin by hitting the Roku Remote Home button. I tried it on a tv series that has AVI's and it just moves on to the next episode.

AVI's plays fine via web interface, writes an FFMPEG log. Pilot.avi plays fine on my production server on a separate computer.

**To Reproduce**

1. Go to Jellyfin Roku
2. Click to play the AVI video, for me, Pilot.avi which is a sole video in a folder
3. Error happens
4. To make it stop, hit the Roku remote home button to exit Jellyfin or else it keeps attempting to open the AVI

**Expected behavior**

Play the video, write the FFMPEG log, finish playing the AVI, stop writing the FFMPEG log, send video finished email notification.

**Logs**

No FFMPEG log is created. See additional context section for a manual log

Server log portion (Note, this repeats over and over again for each attempt until hitting the Roku remotes Home Button
```
[2022-06-11 06:21:21.724 -04:00] [INF] [80] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "tv". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2022-06-11 06:21:21.726 -04:00] [INF] [80] Jellyfin.Api.Helpers.MediaInfoHelper: StreamBuilder.BuildVideoItem( Profile="Anonymous Profile", Path="G:\_TVdir\DVR\Pilot.avi", AudioStreamIndex=0, SubtitleStreamIndex=-1 ) => ( PlayMethod=Transcode, TranscodeReason=VideoCodecNotSupported ) "media:/videos/88e87a67-2387-ea14-8856-20a708545f72/master.m3u8?MediaSourceId=88e87a672387ea14885620a708545f72&VideoCodec=h264&AudioCodec=aac&AudioStreamIndex=0&VideoBitrate=139616000&AudioBitrate=384000&MaxFramerate=47.958286&api_key=<token>&SubtitleMethod=Encode&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=70dd9e21816b61c9a874cfbfdab7f964&SegmentContainer=ts&MinSegments=1&BreakOnNonKeyFrames=True&mpeg4-level=5&mpeg4-videobitdepth=8&mpeg4-profile=advancedsimpleprofile&h264-profile=high,main,baseline,constrainedbaseline&h264-level=51&aac-audiochannels=2&TranscodeReasons=VideoCodecNotSupported"
[2022-06-11 06:21:22.122 -04:00] [INF] [110] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Roku" "1.4.12" playing "Pilot". Stopped at "0" ms
[2022-06-11 06:21:22.733 -04:00] [INF] [61] MediaBrowser.Plugins.SmtpNotifications.Notifier: Sending email "REDACTED"  with subject "Jellyfin: tv has finished playing Pilot on Roku Express (Roku Player)"
[2022-06-11 06:21:22.991 -04:00] [INF] [61] MediaBrowser.Plugins.SmtpNotifications.Notifier: Completed sending email "REDACTED"  with subject "Jellyfin: tv has finished playing Pilot on Roku Express (Roku Player)"
```

**Additional context**

AVI's play fine with my Jellyfin 10.7.0  production machine using the same settings, only on a different server, different Jellyfin version.

Computer:

Device name	library
Processor	Intel(R) Core(TM) i7-4770 CPU @ 3.40GHz   3.40 GHz
Installed RAM	16.0 GB
System type	64-bit operating system, x64-based processor

Windows Spec:
Edition	Windows 10 Pro
Version	21H2
OS build	19044.1706
Experience  Windows Feature Experience Pack 120.2212.4170.0

ffmpeg version 5.0.1-Jellyfin Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 10-win32 (GCC) 20220113

**Testing Procedures**
I ran Pilot.avi through Jellyfin's ffmpeg, version ffmpeg version 5.0.1-Jellyfin,  via command line with a command that I think, based on a previous AVI ffmpeg log before upgrade, that Jellyfin should be doing and it ran corrected: 
```
C:\Users\REDACTED>"C:\Program Files\Jellyfin\Server\ffmpeg.exe" -f avi -i file:"G:\_TVdir\DVR\Pilot.avi" -map_metadata -1 -map_chapters -1 -threads 0 -map 0:1 -map 0:0 -map -0:s -codec:v:0 libx264 -pix_fmt yuv420p -preset veryfast -crf 23 -maxrate 4557570 -bufsize 9115140 -profile:v:0 high -level 41 -x264opts:0 subme=0:me_range=4:rc_lookahead=10:me=dia:no_chroma_me:8x8dct=0:partitions=none  -force_key_frames:0 "expr:gte(t,0+n_forced*3)" -vf "scale=trunc(min(max(iw\,ih*dar)\,min(512\,384*dar))/2)*2:trunc(min(max(iw/dar\,ih)\,min(512/dar\,384))/2)*2" -start_at_zero -vsync -1 -codec:a:0 aac -ac 2 -ab 384000 -copyts -avoid_negative_ts disabled -max_muxing_queue_size 2048 -f hls -max_delay 5000000 -hls_time 3 -hls_segment_type mpegts -start_number 0 -hls_segment_filename "D:\_______________DVRtv\Transcode\124f65798514db00b74ca33f7d6f29ec%d.ts" -hls_playlist_type vod -hls_list_size 0 -y "D:\_______________DVRtv\Transcode\124f65798514db00b74ca33f7d6f29ec.m3u8"
```

**Edit further testing**
I rolled back to 10.8.0 Beta3 that had ffmpeg version "5.0.1" and avi's wouldn't play.

I rolled back to 10.8.0 Beta2, with ffmpeg version 4.4.1-Jellyfin and AVI's started playing again.

I downloaded and installed ffmpeg version 5.0.1-Jellyfin from the repo, to a new folder, c:\ffmpeg, and pointed ffmpeg there and it plays avi's on Beta2 with ffmpeg 5.0.1.

I remember now, the previous problem I had back in the middle of May, with ffmpeg and avi's was with beta3 doing the same thing, which I then rolled back to Beta2. I was intending to do an issue report but time got away.

**Comment by karimw786** [comment]
I have encountered the same bug playing AVIs. I am on Jellyfin server 10.8.1 and the Jellyfin Roku app version 1.4.12. However, if I switch to the 1.4.7 branch of the Roku app, the AVIs play fine. This leads me to believe that it's not the server but rather the Roku app that's the problem.

**Comment by neilsb** [comment]
Can you check this on the new Roku client release (v1.6). I believe it may have been fixed as part of #678

**Comment by p37307** [comment]
> 

I just checked on 2 different Roku clients running v1.6 and the issue is resolved for me.
